### ISSUE TITLE
Caption text is very large on Android devices

### ISSUE THREAD

**Issue by elynema** [samvera-labs/ramp#406]
# Description 

Caption text seems to be excessively large on Android devices (Samsung and Google).

Example:

<img width="600" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOTFJQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--59fd676e4e99f12787dfa898730870de62369eb2/img_4908.jpeg" alt="img_4908.jpeg" />

**Comment by masaball** [comment]
Test captions are on the second item in https://avalon-dev.dlib.indiana.edu/media_objects/g158bh28p, "lunchroom_manners.mp4". This issue was present on all browsers on all Android devices I tested against. The captions are only un-manageably large in portrait full screen mode. Captions are a normal size in portrait mode. Maybe even a bit on the small side. <img width="878" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeTFKQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--7be5e947e17b9258ab14adc2797efea0e627dffd/Screenshot_20240209_160645.png" alt="Screenshot_20240209_160645.png" />

Setting the font size to 50% in the captions settings menu seems to get them to an acceptable size to my eyes: 
<img width="394" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBelpKQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--a7ea2a67588c5ebf307fcb2c1a50da5e5f35d293/Screenshot_20240209_160947.png" alt="Screenshot_20240209_160947.png" />

We may be able to force a specific font size or percentage via CSS for full-screen, non-rotated video. Though forcing it via CSS may prevent the option in the settings menu from working if someone did want to adjust the font. But that's probably acceptable if it means that whole video isn't covered by the captions by default?

**Comment by elynema** [comment]
It looks like I am seeing something similar on iPad in Safari. This is likely because the Safari browser on iPad uses Ramp player in fullscreen, just like Android does?

<img width="525" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBd1ZRQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--94097f70a6d5ec082c304dde3d531e0c3afd65c7/image.png" alt="image.png" />

**Comment by elynema** [comment]
@charumitraravi Can you test this in the Ramp demo site?

**Comment by charumitraravi** [comment]
<img width="557" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNTFiQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--053d3f01b742828f0ae0c05585b147f3d9880ce9/Screenshot%202024-02-14%20at%201.45.03%E2%80%AFPM.png" alt="Screenshot 2024-02-14 at 1.45.03 PM.png" />
QA completed. Issue seems to be resolved in Android. In ipad, when there are double captions (which is a bug) one of the captions still appear to be taking up half of the space ,since it is placed around the center of the video

**Comment by elynema** [comment]
@joncameron @masaball @charumitraravi  This is what I am seeing on Android devices? Does this match what others are seeing? I tested on a Galaxy fold to make sure it wasn't a device we had tested on recently.

<img width="326" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBN3hjQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--93bcb2eb1e98787cfd0fa0c9a4ef1472be4d4110/bs_realdroid_Mobile_Samsung%20Galaxy%20Z%20Fold%205-13.0-1812x2176.jpg" alt="bs_realdroid_Mobile_Samsung Galaxy Z Fold 5-13.0-1812x2176.jpg" />

**Comment by masaball** [comment]
I am seeing the double spacing on Avalon-dev but not on the Ramp demo site so there seems to be some styling on the Avalon side causing this. I can take a look at it today unless you want us to wait on fixing it until next release.

**Comment by elynema** [comment]
This looks good on Android.

I am having a strange experience in iPad. Here's what I see (as this device has captions on in the native player, I'm seeing captions twice).

<img width="559" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeHRvQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--b068d482693f6e653f680b6ca4d755380e4ec355/bs_realios_Tablet_iPad%20Pro%2012.9%202021-17.0%20%282%29.jpg" alt="bs_realios_Tablet_iPad Pro 12.9 2021-17.0 (2).jpg" />

When I turn off captions in Ramp, the smaller set disappears, which implies that the styling is working for the captions provided by IIIF as we intended. However, when I adjust text size for the captions under 'caption settings' in Ramp, the text size for only the larger captions (presumably coming from the native player) change. The other captions that seem to be provided by Ramp don't change in size.

Is this because the CSS is forcing font size for the Ramp-provided captions, so users can't adjust it in the caption settings? It sounded like Mason suggested that may happen in his original research.

If so, then I think this is working correctly and can be closed. @joncameron Do you want another issue opened for the large native player caption size on iPad (Safari)? The native player caption text looks ok on iPhone, so I think this may be potentially an iPad only issue, and only when captions are turned on in the native player, which happens in Chrome rather than Safari, so it might be kind of edge.

**Comment by joncameron** [comment]
Created samvera-labs/ramp#423 for the native player caption issue.
